RELEASE/scripts/autoscend/auto_pre_adv.ash

```diff
@@ -961,6 +970,15 @@ boolean auto_pre_adventure()
 	return true;
 }
 
+//At low max MP, important to keep MP near max, since every saucestorm (etc.) counts
+void low_mp_handler()
```

Comment:
You can just move this above & not need the empty placeholder function.

Reply:
Mischief managed with _Improve low_mp_handler readability_
Will need a few days for my hardcore run to finish so I can test again